aws/aws_tests.go

```diff
@@ -379,65 +379,121 @@ func containersRuntimeValidator(region string, isFargate bool) func(t *testing.T
 			t.Logf("GET %v [%v/%v]: %v", baseURL+"custom", resp.StatusCode, contentType, string(bytes))
 		}
 
-		// Wait for five minutes before getting logs.
-		time.Sleep(5 * time.Minute)
```

Comment:
a nice benefit of the new approach is that if logs come sooner, we can do our checks the moment they're available.  for example, in this PR that saved 4minutes :)